### PR TITLE
Covered the 4.0 release of Ember Sinon QUnit

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

File: source/2019-09-13-the-ember-times-issue-115.md [hunk 1]
```diff
@@ -74,13 +75,13 @@ Be sure to check out [the full video](https://www.instagram.com/tv/B2CCM60AOZG/?
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Jordan Hawker (@elwayman02)](https://github.com/elwayman02) and [Steve Calvert (@scalvert)](https://github.com/scalvert) teamed up to bring us the best [Sinon.js](https://sinonjs.org/) experience possible. 💞
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Thanks to their work, [ember-sinon-qunit v4](https://github.com/elwayman02/ember-sinon-qunit) allows us to do a one-time setup, use the `sinon` object directly in our tests, and never worry about cleaning up mocks after the tests are run. To learn more about the motivation for change and the new syntax, you can check out [Jordan's blog post](https://www.jordanhawker.com/p/187541610821) and the [addon's README file](https://github.com/elwayman02/ember-sinon-qunit#usage).
+
+If you have been using Steve's [ember-sinon-sandbox](https://github.com/scalvert/ember-sinon-sandbox) or [ember-sinon-sinoff](https://github.com/scalvert/ember-sinon-sinoff), please check the [README file](https://github.com/elwayman02/ember-sinon-qunit#migrating-to-ember-sinon-qunit) to decide your migration path.
```
Comment (jayjayjpg) [inline source/2019-09-13-the-ember-times-issue-115.md]: I'm not exactly sure how this works in English, but would it be either

```suggestion
If you have been using Steve's [ember-sinon-sandbox](https://github.com/scalvert/ember-sinon-sandbox) or [ember-sinon-sinoff](https://github.com/scalvert/ember-sinon-sinoff), please check the [README file](https://github.com/elwayman02/ember-sinon-qunit#migrating-to-ember-sinon-qunit) to decide on your migration path.
```

or....

Comment (jayjayjpg) [inline source/2019-09-13-the-ember-times-issue-115.md]: ...something like this?

```suggestion
If you have been using Steve's [ember-sinon-sandbox](https://github.com/scalvert/ember-sinon-sandbox) or [ember-sinon-sinoff](https://github.com/scalvert/ember-sinon-sinoff), please check the [README file](https://github.com/elwayman02/ember-sinon-qunit#migrating-to-ember-sinon-qunit) to choose your migration path.
```

What do you think?

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more question about a phrasing, but apart from that this is already good to go - thank you for another :sparkles: writeup this week!

File: source/2019-09-13-the-ember-times-issue-115.md (hunk 1 above)
Comment (ijlee2) [reply source/2019-09-13-the-ember-times-issue-115.md]: Hmm, I'm not sure myself, haha. Maybe both?

Let's go with "choose" to be safe. :)

**Pull request merged**